.travis.yml

```diff
@@ -224,12 +224,3 @@ matrix:
         submodules: false # avoid cloning ethereum/tests
       script:
         - go run build/ci.go purge -store gethstore/builds -days 14
-
```

Comment:
Please remove all the `repo = ethereum/go-ethereum AND ` stuff from the Travis rules that were added to support ethersphere.